azerothcore/azerothcore-wotlk
# fix(DB): Blackwing Technician spawn and addon

## Changes Proposed:
- Added spawns based on sniffs and updated addon for their emote.
- Cleared spawns.

## Issues Addressed:

- Closes not reported issue

## SOURCE:

Brotalnia's sniffs

## Tests Performed:

- Tested in-game for DB errors


## How to Test the Changes:


1. Apply SQL and check for errors
2. They should have the crafting emote playing

## Known Issues and TODO List:


None


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
